portality/static/widget/simple_search.js

```diff
@@ -37,9 +37,31 @@ function scriptLoadHandler() {
 
 /******** Our main function ********/
 function main() { 
-    jQuery(document).ready(function($) { 
-        $('#doaj-simple-search-widget').load("https://doaj.org/static/widget/simple_search_body.html");
+    jQuery(document).ready(function($) {
+
+        // doaj_url values:
+        // dev: 'http://localhost:5004'
+        // test: 'https://testdoaj.cottagelabs.com/'
+        // production: 'https://www.doaj.org'
+        let doaj_url = 'http://localhost:5004';
+
+        let head = document.head || document.getElementsByTagName('head')[0];
+
+        let scr  = document.createElement('script');
+        scr.src = doaj_url + '/static/vendor/feather/feather.min.js';
+        scr.async = false;
+        scr.defer = false;
+        head.insertBefore(scr, head.firstChild);
+
+        scr = document.createElement('link');
+        scr.rel = 'stylesheet';
+        scr.href = doaj_url + '/static_content/_site/css/main.css';
```

Comment:
This URL has changed with https://github.com/DOAJ/doaj/pull/1880

Reply:
Stylesheets and classes have also been moved around -- possible to rebase this branch off of this one? https://github.com/DOAJ/doaj/pull/1880  It will be much easier to style (which should be pretty minimal)